templates/zerver/help/set-notifications-for-a-single-stream.md

```diff
@@ -0,0 +1,20 @@
+# Set notifications for a single stream
+
+If you would like to have different notifications for different streams,
```

Comment:
I think you mean "You can have different notification settings for different streams." or something like that.